_static/demos/circuits_as_fourier_series/sketch.js

```diff
@@ -0,0 +1,1035 @@
+var url_prefix = '../../_static/demos/circuits_as_fourier_series/src/';
+
+fetch(url_prefix + 'fourier0-1.png', {method: 'HEAD'}).then(res => {
+    if (!res.ok) {
+        url_prefix = '../_static/demos/circuits_as_fourier_series/src/';
+    }
+});
```

Comment:
```suggestion
var url_prefix = '../_static/demos/circuits_as_fourier_series/src/';

// fetch(url_prefix + 'fourier0-1.png', {method: 'HEAD'}).then(res => {
//     if (!res.ok) {
//         url_prefix = '../_static/demos/circuits_as_fourier_series/src/';
//     }
// });
```

Hey @branedrane, this fix should allow the p5js images to work correctly on this PR. However, you'll need to undo this change before we merge it in :)